cms/pom.xml

```diff
@@ -194,6 +194,12 @@
             <groupId>org.onehippo.forge.folderctxmenus</groupId>
             <artifactId>folderctxmenus-frontend</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>1.1.1</version>
+        </dependency>
```

Comment:
Following our discussion and comment against `GeoJsonMapParser` class - let's remove this dependency.